### PR TITLE
Added control specs for App service controls

### PR DESCRIPTION
**Pull request by TarunKrShukla**

### CODE REVIEW

File: Control coverage/Feature/AppService.md [hunk 1]
```diff
@@ -2,5 +2,8 @@
...
...:</b><br>Use Approved TLS Version in App Service.<br><b>Description: </b><br> Use approved version of TLS for the App Service. |<b> ARM API to get configuration of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/web? <br> api-version=2018-11-01 <br><b>Properties:</b><br> properties/minTlsVersion <br><br> <b> ARM API to list all deployment slots of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots? <br> api-version=2019-08-01 <br><b>Properties:</b><br> name <br><br> <b> ARM API to get configuration of an App Service slot: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slotName}/config/web? <br> api-version=2019-08-01 <br><b>Properties:</b><br> properties/minTlsVersion| <b>Passed: </b><br>Current minimum TLS version is set to either equal or greater than the required min TLS version for all the slots in App Service.<br><b>Failed: </b><br>Current minimum TLS version is less than required min TLS version for any of the slot in App Service. |
```
Comment (gvaradarajan-msft) [inline Control coverage/Feature/AppService.md:6]: Any reason why "ARM API to list all the App Services in a subscription" is not listed for all Controls, except `Azure_AppService_DP_Dont_Allow_HTTP_Access`? The App Service name from this call is used in any subsequent calls for the same App Service.

File: Control coverage/Feature/AppService.md [hunk 2]
```diff
@@ -2,5 +2,8 @@
...
\ No newline at end of file
+| <b>ControlId:</b><br>Azure_AppService_DP_Dont_Allow_HTTP_Access<br><b>DisplayName:</b><br>Use HTTPS for app services.<br><b>Description: </b><br> App Service must only be accessible over HTTPS. |<b> ARM API to list all the App Services in a subscription: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.Web/sites? <br> 2018-11-01 <br><b>Properties:</b><br> properties/httpsOnly <br><br> <b> ARM API to list all deployment slots of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots? <br> api-version=2019-08-01 <br><b>Properties:</b><br> properties/httpsOnly| <b>Passed: </b><br>HTTPS Only setting is eanbled for all slots in App Service.<br><b>Failed: </b><br>HTTPS Only setting is disabled for any slot in App Service. |
```
Comment (gvaradarajan-msft) [inline Control coverage/Feature/AppService.md:5]: Typo in "Only setting is **eanbled** for all slots in App Service.". **eanbled** to be corrected to **enabled**.

File: Control coverage/Feature/AppService.md [hunk 3]
```diff
@@ -2,5 +2,8 @@
...
... App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/web?<br> api-version=2018-11-01 <br><b>Properties:</b><br> properties/detailedErrorLoggingEnabled, properties/httpLoggingEnabled, properties/requestTracingEnabled  <br><br> <b> ARM API to list all deployment slots of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots? <br> api-version=2019-08-01 <br><b>Properties:</b><br> name <br><br> <b> ARM API to get configuration of an App Service slot: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slotName}/config/web? <br> api-version=2019-08-01 <br><b>Properties:</b><br> properties/detailedErrorLoggingEnabled, properties/httpLoggingEnabled, properties/requestTracingEnabled| <b>Passed: </b><br>HTTP Logging, DetailedError Logging and Request Tracing, all three must be eanbled for all slots in App Service.<br><b>Failed: </b><br>HTTP Logging, DetailedError Logging and Request Tracing, any one of the three logging is disabled for any slot of App Service. |
```
Comment (gvaradarajan-msft) [inline Control coverage/Feature/AppService.md:9]: Typo in "HTTP Logging, DetailedError Logging and Request Tracing, all three must be **eanbled** for all slots in App Service.". **eanbled** to be corrected to **enabled**.

**Review by gvaradarajan-msft — CHANGES_REQUESTED** [review]

File: Control coverage/Feature/AppService.md (hunk 3 above)
Comment (TarunKrShukla) [reply Control coverage/Feature/AppService.md]: done, fixed the typo.

File: Control coverage/Feature/AppService.md (hunk 2 above)
Comment (TarunKrShukla) [reply Control coverage/Feature/AppService.md]: done, fixed the typo.

File: Control coverage/Feature/AppService.md (hunk 1 above)
Comment (gvaradarajan-msft) [reply Control coverage/Feature/AppService.md:6]: This is not required post our offline conversation.

**Review by gvaradarajan-msft — APPROVED** [review]

**Pull request merged**